### PR TITLE
Create reagent vomicilin

### PR DESCRIPTION
**Pull request by Varelion**
## What Does This PR Do

This PR creates a reagent 'vomicilin' and deploys it where other harmless reagents are present. Vomicilin is a reference to the U.S. sitcom 'The Office' seen in Episode no: Season 2; Episode 11.

## Why It's Good For The Game
It adds a drug that induces vomiting, doing a small clear of toxin damage at the cost of nutrients.

## Testing
Launched the client, and tested creation and use of vomicilin.

## Changelog
:cl:
add: Added reagent vomicilin
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 1]
```diff
@@ -429,6 +429,28 @@
...
+		var/update_flags = STATUS_UPDATE_NONE
+		update_flags |= M.adjustToxLoss(-0.7*REAGENTS_EFFECT_MULTIPLIER, FALSE)
+		M.fakevomit(1)
+		M.adjust_nutrition(-2)
+	else
+		var/update_flags = STATUS_UPDATE_NONE
+		update_flags |= M.adjustToxLoss(-0.5*REAGENTS_EFFECT_MULTIPLIER, FALSE)
+		M.fakevomit(0)
+		M.adjust_nutrition(-1)
+
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/reagents/medicine.dm:453]:
```suggestion
	..()
	
```

**Review by DGamerL — CHANGES_REQUESTED** [review]
Always call parent here

**Comment by Mitchs98** [comment]
This is essentially toilet water 2, but slightly harder to acquire. If it's to exist it really needs more downsides than just minor amounts of nutrition per vomit. Toxins like toilet water for example. I get it's meant to be a reference but in terms of the game it just reads as an alternative to toilet water that doesn't deal damage to you.

**Comment by matttheficus** [comment]
Please dear god I dont want more vomit spam - toilet water is bad enough already and the idea of a chem that exists only for a reference is kinda bleh.

**Comment by matttheficus** [comment]
However, if this chemical was used as a sorta "on the fly dialysis" of some sort, I might see some merit?

But as-is, not a fan at all.

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 2]
```diff
@@ -429,6 +429,29 @@
...
+		var/update_flags = STATUS_UPDATE_NONE
+		update_flags |= M.adjustToxLoss(-0.7*REAGENTS_EFFECT_MULTIPLIER, FALSE)
+		M.fakevomit(1)
+		M.adjust_nutrition(-2)
+	else
+		var/update_flags = STATUS_UPDATE_NONE
+		update_flags |= M.adjustToxLoss(-0.5*REAGENTS_EFFECT_MULTIPLIER, FALSE)
+		M.fakevomit(0)
+		M.adjust_nutrition(-1)
+	return ..() | update_flags
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/reagents/medicine.dm:453]:
```suggestion
	return ..()
```
Whoops

**Comment by TravisAngeI** [comment]
Not to dogpile on the already present complaints, but why is your Why Is It Good For The Game just an explanation of what the chem does rather than why it should be added

File: code/modules/reagents/chemistry/reagents/medicine.dm (hunk 2 above)
Comment (farie82) [reply code/modules/reagents/chemistry/reagents/medicine.dm:453]: The flags should be included. But the var should be declared above the if(prob)

**Comment by farie82** [comment]
The design team has voted against this PR to be merged in its current form. For the following reasons (beside those posted on the PR itself already)
- Just not a fan of having it implemented in any way tbh.
- We do not need a reagent for vomiting, not after toilet water

You can perhaps talk with the design team to tweak the PR if you want to get this merged in another form. For now I'll close the PR in its current form.

**Pull request closed without merging**